zenoh/src/publication.rs

```diff
@@ -536,6 +538,7 @@ impl<'a, 'b> PublisherBuilder<'a, 'b> {
 
     /// Restrict the matching subscribers that will receive the published data
     /// to the ones that have the given [`Locality`](crate::prelude::Locality).
+    #[cfg(feature = "unstable")]
```

Comment:
`Locality` should be only on subscriber and queryable side for the time being at least.